js/cljdoc.js

```diff
@@ -126,6 +126,18 @@ function restoreSidebarScrollPos() {
   localStorage.removeItem("sidebarScrollPos");
 }
 
+if (document.querySelector(".main-scroll-view")) {
+  document.getElementById("meta-icon").onclick = function() {
+    document.getElementById("meta-icon").style.display = "none";
+    document.getElementById("meta-dialog").style.display = "block";
+  };
+
+  document.getElementById("close").onclick = function() {
+    document.getElementById("meta-dialog").style.display = "none";
+    document.getElementById("meta-icon").style.display = "block";
```

Comment:
Instead of using `style.display` directly here we could also use tachyons classes which would also allow us to apply media queries — which would be useful if we decide to not show this in smaller screens for now.

For instance using an elements [`classList`](https://developer.mozilla.org/en-US/docs/Web/API/Element/classList) property we could do this:

```js
// to show it:
document.getElementById("meta-icon").classList.replace("dn", "db-ns");
// to hide it: 
document.getElementById("meta-icon").classList.replace("db-ns", "dn");
```

`db-ns` means "**d**isplay: **b**lock if screen **n**ot **s**mall", `dn` means "**d**isplay: **n**one" (and because there's no qualifier like `-ns` it applies to all screen sizes).

Reply:
that's super handy, thanks for the suggestion! I'll make that change 👍